moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1015,6 +1068,57 @@ bool MoveItConfigData::inputKinematicsYAML(const std::string& file_path)
   return true;  // file created successfully
 }
 
+// ******************************************************************************************
+// Input ros_controllers.yaml file
+// ******************************************************************************************
+bool MoveItConfigData::inputROSControllersYAML(const std::string& file_path)
```

Comment:
Since its a `gui` code, Well there is two options here: first the fast one
>Add move the `addDefaultControllers` function to `moveit_data_config`, by that I'll dodge setting up Qtests.

Second
>Setup QT tests

I think I should do the second, it might delay this PR a bit, but setting up QT tests will come in handy later with other parts of the `setup_assistant`.

Reply:
The moveit_config_data class does (should) not use QT, its in the /tools directory, not in the /widgets directory. It should be a set of easily testable libraries. We don't need QT tests, that's sounds too complicated

Reply:
Okay I'll look into testing it without having to use the QT test, or the moving anything to the `moveit_config_data`.